lib/hutch.rb

```diff
@@ -18,6 +17,10 @@ def self.register_consumer(consumer)
     self.consumers << consumer
   end
 
+  def self.server?
```

Comment:
You can run Hutch just for publishing and still use it in a server of some kind. A more suitable name would be `daemon?` or `consumer_service?` (mind you, neither is perfect).